openshift/templates/README.md

```diff
@@ -32,7 +32,7 @@ Create [build images](./images/README.md)
 | alpine | 3.15.4 | Base Alpine linux |
 | mariadb | 10.6.8-r0 | MariaDB version that gets build in base-images via Dockerfile. Dependent on the [Alpine version](https://pkgs.alpinelinux.org/packages?name=mariadb&branch=v3.15): |
 | nginx | 1.23.1-alpine | Nginx web server, used to serve WordPress / PHP |
-| wordpress | 6.0.2-php7.4-fpm-alpine | PHP-FPM for WordPress |
+| wordpress | 6.0.3-php7.4-fpm-alpine | PHP-FPM for WordPress |
```

Comment:
i know that we said this was 6.0.3, but should it be 6.1

Reply:
Thanks for raising this. We have decided to leave this at 6.0.3 for now as an exercise.